vars/pipelineStagnetDeploy.groovy

```diff
@@ -140,8 +139,7 @@ void call() {
                                     // Note: environment variables PSSH_KEYFILE and PSSH_USER
                                     //        are set by withCredentials wrapper
                                     sh label: 'ansible deploy run', script: """#!/bin/bash -e
-                                        export ANSIBLE_FORCE_COLOR=true
-                                        ANSIBLE_HOST_KEY_CHECKING=False ansible-playbook \
```

Comment:
Are you sure We can remove the `ANSIBLE_HOST_KEY_CHECKING` It was failing when new Jenkins agent has been started and no servers signatures are added to the ~/.ssh/known_hosts